lib/active_merchant/billing/gateways/reach.rb

```diff
@@ -0,0 +1,176 @@
+require 'base64'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class ReachGateway < Gateway
+      # TODO: Things to check
+      # * The list of three digit fractions but only accept 2
+      # * Not sure the list of countries and currencies
+
+      self.test_url = 'https://checkout.rch.how/'
+      self.live_url = 'https://checkout.rch.io/'
+
+      self.supported_countries = ['US']
+      self.default_currency = 'USD'
+      self.supported_cardtypes = %i[visa diners_club american_express jcb master discover maestro]
+
+      self.homepage_url = 'https://www.withreach.com/'
+      self.display_name = 'Reach'
+      self.currencies_without_fractions = %w(BIF BYR CLF CLP CVE DJF GNF ISK JPY KMF KRW PYG RWF UGX UYI VND VUV XAF XOF XPF IDR MGA MRO)
+
+      API_VERSION = 'v2.21'.freeze
+      STANDARD_ERROR_CODE_MAPPING = {}
+      PAYMENT_METHOD_MAP = {
+        american_express: 'AMEX',
+        cabal: 'CABAL',
+        check: 'ACH',
+        dankort: 'DANKORT',
+        diners_club: 'DINERS',
+        discover: 'DISC',
+        elo: 'ELO',
+        jcb: 'JCB',
+        maestro: 'MAESTRO',
+        master: 'MC',
+        naranja: 'NARANJA',
+        visa: 'VISA'
+      }
+
+      def initialize(options = {})
+        requires!(options, :merchant_id, :secret)
+        super
+      end
+
+      def authorize(money, payment, options = {})
+        request = build_checkout_request(money, payment, options)
+        add_customer_data(request, options, payment)
+
+        post = { request: request, card: add_payment(payment) }
+        commit('checkout', post)
+      end
+
+      def purchase(money, payment, options = {})
+        options[:capture] = true
+        authorize(money, payment, options)
+      end
+
+      def capture(money, authorization, options = {})
+        merchant_id, order_id = authorization.split '#'
+        post = { request: { MerchantId: merchant_id, OrderId: order_id } }
+
+        commit('capture', post)
+      end
+
+      def supports_scrubbing?
+        true
```

Comment:
It doesn't look like it...

Reply:
mm @curiousepic I left the method there because the work to add the scrubbing is in another story and is almos done, do you think is necessary to remote that line from here and add it on that PR ?

Reply:
Technically that would be proper, if that piece will be merged in a separate commit.

Reply:
ok I'm going to remove it 👍